lore/end_users/status/README.md

```diff
@@ -40,3 +46,7 @@ The Zuul Pipelines counter provides an easy look at how many pipelines exist in
 ### Service Status
 
 Nodepool, Zookeeper, and Zuul are three core services that are essential to the stability of a BonnyCI cluster. If anything except the green "OK" message is displayed for these service statuses, then BonnyCI could be experiencing a full service outage.
+
+## Build Status
+
+The [Build Status page](https://travis-ci.org/BonnyCI) links to the Travis-CI page with our repositories. Yes we see the humor in a CI tool using a different CI tool for testing. This is a temporary home for us.
```

Comment:
@j2sol: To be clear, I am not talking about a historical view (although, yes, Travis-CI does offer this). We are currently linking to the Travis build status page. All it does is show the current status of the repos (passing, failing, error, or no data).

I am not trying to argue for a historical view, although I disagree with your assertion about their value. Being able to view recent results can be very useful if trying to trace when an issue began (if, for example, a test did not catch an issue).